src/main/java/mods/railcraft/api/carts/IEnergyTransfer.java

```diff
@@ -1,92 +1,92 @@
 /*------------------------------------------------------------------------------
- Copyright (c) CovertJaguar, 2011-2020
+Copyright (c) CovertJaguar, 2011-2020
 
- This work (the API) is licensed under the "MIT" License,
- see LICENSE.md for details.
- -----------------------------------------------------------------------------*/
+This work (the API) is licensed under the "MIT" License,
+see LICENSE.md for details.
+-----------------------------------------------------------------------------*/
 
 package mods.railcraft.api.carts;
 
 /**
- * This interface is implemented by the Energy Cart
- * and is used by the Energy Loaders to charge/discharge carts.
- * It is roughly equivalent to the IItemTransfer interface
- * and based on ElectricItem and IElectricItem.
- *
- * This interface has been superseded by the CapabilityCartCharge
- * interface for general use. It remains in use solely for the
- * IC2 based Energy Loaders.
- *
- * @author CovertJaguar <https://www.railcraft.info>
- */
+* This interface is implemented by the Energy Cart
+* and is used by the Energy Loaders to charge/discharge carts.
+* It is roughly equivalent to the IItemTransfer interface
+* and based on ElectricItem and IElectricItem.
+*
+* <p>This interface has been superseded by the CapabilityCartCharge
+* interface for general use. It remains in use solely for the
+* IC2 based Energy Loaders.
+*
+* @author CovertJaguar (https://www.railcraft.info)
+*/
 public interface IEnergyTransfer {
```

Comment:
Is this needed, can't we just use Forge's `IEnergyStorage`?

Reply:
we can, but how would it interact with charge?